src/Misc/brdfTextureTools.ts

```diff
@@ -62,7 +62,6 @@ export class BRDFTextureTools {
 
                 // Cleanup
                 engine.restoreDefaultFramebuffer();
-                engine._releaseTexture(texture);

```

Comment:
@sebavan do you know why the texture was being released here as it looks like its still used (Needed when contextRestore is called). Was another texture meant to be disposed of instead?

Reply:
I think it is swapped earlier isnt it ? it might also be a bug, you could try forcing a context lost to check ?

Reply:
The expanded texture should then be in use if expanded. not releasing the original might cause a leak.

Reply:
Oh I think I see, engine._releaseTexture(texture); causes the texture to be removed from the cache but then expandedTexture._swapAndDie(texture); is called but texture never gets added back to the texture cache to be restored on context loss. Ill give that a try, thanks